types/stripe/index.d.ts

```diff
@@ -8499,15 +8595,19 @@ declare namespace Stripe {
              * ID of the default payment method for the subscription. It must belong to the customer associated with the subscription.
              * If not set, invoices will use the default payment method in the customer’s invoice settings. [Expandable]
              */
-            default_payment_method: string | null;
+            default_payment_method: string | paymentMethods.IPaymentMethod | null;
 
             /**
              * ID of the default payment source for the subscription.
              * It must belong to the customer associated with the subscription and be in a chargeable state.
              * If not set, defaults to the customer’s default source. [Expandable]
              */
             default_source: string | null;
-
+            /**
```

Comment:
And white space here.

Reply:
@pushplay Ok done, I do not know if I agree with this whitespace rule though. 
It is not enforced by the linter and the default prettier behaviour is: Collapse multiple blank lines to one blank line and otherwise leave the line breaks alone.

Having this rule on top of that, just seems to slow down the process.

what do you think @vaskevich ?